app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  def latest_posts
+    Post.where(author: self).limit(3)
+  end
```

Comment:
- [ ] Setting associations between models help you call each other from each other so that you don't need to call the class itself. A user has many `posts`, therefore you would call his 3 recent posts by using that association.
hint:
```rb
  posts.order('created_at Desc').limit(3)
```